scRNA-seq-advanced/setup/rms/.gitignore

```diff
@@ -0,0 +1,5 @@
+# ignore the local data used for testing directories
```

Comment:
Guessing this file isn't needed anymore?

Reply:
You are correct!